spec/support/test_helpers/candidate_helper.rb

```diff
@@ -1,6 +1,9 @@
 module CandidateHelper
-  def create_and_sign_in_candidate
-    candidate = create(:candidate)
+  def create_and_sign_in_candidate(email_address: nil)
+    args = {}
```

Comment:
Instead of passing down the email, we can also remember the currently sign in user and use the email via that in the test. Similar to how we have a `currently_authenticated_provider` in the API specs:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/899a6b45ab0bf32e7eb9314e56d56a13f42645d1/spec/support/vendor_api/vendor_api_spec_helpers.rb#L19-L26

Reply:
I like the idea 👍

Reply:
In my case, I'll call the method:
```
  def current_candidate
    @current_candidate ||= create(:candidate)
  end
```
since the first time you call the method, candidate is not authenticated, and this:
`login(currently_authenticated_candidate)`
would not sound right
`current_candidate` keeps the options open, it says nothing about authentication.